WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -237,25 +234,16 @@ def getQueues( self, resourceDict ):
             si00 = float( self.queueDict[queueName]['ParametersDict']['SI00'] )
             queueCPUTime = 60. / 250. * maxCPUTime * si00
             self.queueDict[queueName]['ParametersDict']['CPUTime'] = int( queueCPUTime )
-          queueTags = self.queueDict[queueName]['ParametersDict'].get( 'Tag' )
-          if queueTags and isinstance( queueTags, basestring ):
-            queueTags = fromChar( queueTags )
-            self.queueDict[queueName]['ParametersDict']['Tag'] = queueTags
-          if ceTags:
-            if queueTags:
-              allTags = list( set( ceTags + queueTags ) )
-              self.queueDict[queueName]['ParametersDict']['Tag'] = allTags
-            else:
-              self.queueDict[queueName]['ParametersDict']['Tag'] = ceTags
 
           maxMemory = self.queueDict[queueName]['ParametersDict'].get( 'MaxRAM', None )
+
           if maxMemory:
             # MaxRAM value is supposed to be in MB
             maxMemoryList = range( 1, int( maxMemory )/1000 + 1 )
             memoryTags = [ '%dGB' % mem for mem in maxMemoryList ]
             if memoryTags:
               self.queueDict[queueName]['ParametersDict'].setdefault( 'Tag', [] )
-              self.queueDict[queueName]['ParametersDict']['Tag'] += memoryTags
+              self.queueDict[queueName]['ParametersDict']['Tag'] += ','.join( memoryTags )
```

Comment:
This is a misunderstanding. What we agreed about is that SiteDirector will not pass the Tags as an argument of the pilot. The tags are needed for the SiteDirector logic because they are used in the evaluation of availability of payload before submitting pilots to a particular CE/queue. So, I suggest that you remove the SiteDirector changes from this PR and I will add the suppression of the Tag pilot argument in my PR for the SiteDirector ( different problem ) to avoid conflicts